### PR TITLE
dev1 - README

### PR DESCRIPTION
**Pull request by regisxx**
What's your opinion?

### CODE REVIEW

File: README.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 Let's start
+Modification dev1
```
Comment (Caroline00fr) [inline README.txt:2]: What do you mean?

Comment (regisxx) [reply README.txt:2]: Nothing

Comment (Caroline00fr) [reply README.txt:2]: ok then

Comment (Caroline00fr) [reply README.txt:2]: well

**Review by Caroline00fr — APPROVED** [review]
go

**Pull request merged**